itest/litd_accounts_test.go

```diff
@@ -416,21 +416,28 @@ func payNode(invoiceCtx, paymentCtx context.Context, t *harnessTest,
 	stream, err := from.SendPaymentV2(paymentCtx, sendReq)
 	require.NoError(t.t, err)
 
-	result, err := getPaymentResult(stream)
+	result, err := getPaymentResult(stream, false)
 	require.NoError(t.t, err)
 	require.Equal(t.t, result.Status, lnrpc.Payment_SUCCEEDED)
 }
 
-func getPaymentResult(stream routerrpc.Router_SendPaymentV2Client) (
-	*lnrpc.Payment, error) {
+func getPaymentResult(stream routerrpc.Router_SendPaymentV2Client,
```

Comment:
Ah, I did something similar in 9c16e898. But this looks better, feel free to replace the `getFinalPaymentResult` with this flag when rebasing.